src/app/auth/pages/login-page/login-page.component.html

```diff
@@ -4,28 +4,25 @@ <h2 class="login__title">Login</h2>
     <div
         class="login__control"
         [ngClass]="{
-            invalid: form.get('login')?.touched && form.get('login')?.invalid
+            invalid: form.get('email')?.touched && form.get('email')?.invalid
```

Comment:
here is better to create a getter (if value is used more than 2 times)
```
get email() {
    return this.form.get('email');
  }
```

Reply:
*please, fix it in the whole file

Reply:
Fixed.
```
    get email() {
        return this.form.get('email');
    }

    get password() {
        return this.form.get('password');
    }


```
```
[ngClass]="{ invalid: email?.touched && email?.invalid }"